docs/domain/document/index.md

```diff
@@ -1,5 +1,4 @@
 (document)=
-(object)=
```

Comment:
Did you drop the `object` reference by accident?

Reply:
Ah, it is probably because there has been introduced a file called `object.md`, which led to collisions, right? Well, if we rename the file, this reference can stay.